### DISCUSSION TITLE
Drag mouse to highlight and copy text

### DISCUSSION THREAD

**Discussion by hariria** [code-hike/codehike#414] [Ideas and feedback]
Is there any way to make it so that I can drag my mouse and highlight a selection of text in the CodeHike preview and copy it? It seems like it's not possible in any of the examples...

https://github.com/code-hike/codehike/assets/31301117/06628880-bcba-42a4-92bc-bcc0ef0da41d

**Comment by pomber** [comment]
can you try with another browser just in case? what version of Chrome are you using? have you tried other examples? is it just codehike codeblocks that you cant select? I've just tried that same example in Chrome and Firefox and both worked

**Reply by hariria** [reply]
I'm using Chrome 120, MacOS Sonoma and the same behavior occurs with all Codehike examples... not sure why. I just tried on Safari and it worked! Weird that it doesn't work on Chrome. I also tried on an Incognito window to make sure there weren't any preferences or extensions that were impacting the behavior and it still wasn't working.
